src/components/transaction/TransactionTable/index.tsx

```diff
@@ -0,0 +1,295 @@
+  }
+  const limitPriceSettled = getLimitPrice(transaction, isPriceInverted)
+
+  return (
+    <tr key={txHash}>
+      <td>
+        <HeaderTitle>
+          Order ID <HelpTooltip tooltip={tooltip.tradeID} />
+        </HeaderTitle>
+        <HeaderValue>
+          <RowWithCopyButton
+            className="span-copybtn-wrap"
+            textToCopy={uid}
+            contentsToDisplay={
+              <LinkWithPrefixNetwork to={`/tx/${transaction.txHash}`} rel="noopener noreferrer" target="_blank">
+                {getShortOrderId(shortId)}
+              </LinkWithPrefixNetwork>
+            }
+          />
+        </HeaderValue>
+      </td>
+      <td>
+        <HeaderTitle>Type</HeaderTitle>
+        <span className="header-value">
+          <TradeOrderType kind={kind || 'sell'} />
+        </span>
+      </td>
+      <td>
+        <HeaderTitle>Sell Amount</HeaderTitle>
+        <HeaderValue>
+          {renderSpinnerWhenNoValue(sellFormattedAmount) || (
+            <TextWithTooltip textInTooltip={`${sellFormattedAmount} ${sellTokenSymbol}`}>
+              {formattedAmount(sellToken, sellAmount, FormatAmountPrecision.highPrecision)}{' '}
+              {sellToken && network && <TokenDisplay showAbbreviated erc20={sellToken} network={network} />}
+            </TextWithTooltip>
+          )}
+        </HeaderValue>
+      </td>
+      <td>
+        <HeaderTitle>Buy amount</HeaderTitle>
+        <HeaderValue>
+          {renderSpinnerWhenNoValue(buyFormattedAmount) || (
+            <TextWithTooltip textInTooltip={`${buyFormattedAmount} ${buyTokenSymbol}`}>
+              {formattedAmount(buyToken, buyAmount, FormatAmountPrecision.highPrecision)}{' '}
+              {buyToken && network && <TokenDisplay showAbbreviated erc20={buyToken} network={network} />}
+            </TextWithTooltip>
+          )}
+        </HeaderValue>
+      </td>
+      <td>
+        <HeaderTitle>
+          Limit price
+          <Icon icon={faExchangeAlt} onClick={invertLimitPrice} />
+        </HeaderTitle>
+        <HeaderValue>{renderSpinnerWhenNoValue(limitPriceSettled) || limitPriceSettled}</HeaderValue>
+      </td>
+      <td>
+        <HeaderTitle>Created</HeaderTitle>
+        <HeaderValue>
+          <DateDisplay date={expirationDate} showIcon={true} />
+        </HeaderValue>
+      </td>
+      <td>
+        <HeaderTitle>Status</HeaderTitle>
+        <HeaderValue>
+          <StatusLabel status={transaction.status} partiallyFilled={partiallyFilled} />
+        </HeaderValue>
+      </td>
+    </tr>
+  )
+}
+
+const TransactionTable: React.FC<Props> = (props) => {
+  const { transactions, showBorderTable = false } = props
+  const [isPriceInverted, setIsPriceInverted] = useState(false)
+  useEffect(() => {
+    setIsPriceInverted(isPriceInverted)
+  }, [isPriceInverted])
+  const invertLimitPrice = (): void => {
+    setIsPriceInverted((previousValue) => !previousValue)
+  }
+
+  const transactionItems = (items: Order[]): JSX.Element => {
```

Comment:
Can this function be defined outside, to avoid re-instantiating at every render?
AFAICT, it doesn't need anything from the context so it could be moved, but I might be wrong.